webpack/components/extensions/HostDetails/Tabs/TracesTab.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useCallback } from 'react';
+import { Skeleton, Button } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { TableVariant, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import { useSelector } from 'react-redux';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import TableWrapper from '../../../Table/TableWrapper';
+import getHostTraces from './HostTracesActions';
+import { selectHostTracesStatus } from './HostTracesSelectors';
+import './TracesTab.scss';
+
+const TracesTab = () => {
+  const [searchQuery, updateSearchQuery] = useState('');
+  const hostDetails = useSelector(state => selectAPIResponse(state, 'HOST_DETAILS'));
+  const { id: hostId } = hostDetails;
+  const emptyContentTitle = __('This host currently does not have traces.');
+  const emptyContentBody = __('Add traces by installing katello-host-tools-tracer, and applying updates.');
+  const emptySearchTitle = __('No matching traces found');
+  const emptySearchBody = __('Try changing your search settings.');
+  const actionButtons = <Button variant="danger" isDisabled> Restart app </Button>;
+  const fetchItems = useCallback(
+    params =>
+      (hostId ? getHostTraces(hostId, params) : null),
+    [hostId],
+  );
+  const response = useSelector(state => selectAPIResponse(state, 'HOST_TRACES'));
+  const { results, ...meta } = response;
+  const status = useSelector(state => selectHostTracesStatus(state));
+  if (!hostId) return <Skeleton />;
+  /* eslint-disable max-len */
+  return (
+    <div id="traces-tab">
+      <h3>Tracer helps administrators identify applications that need to be restarted after a system is patched.</h3>
```

Comment:
`__() `<== you need to add translation to any places users will see your strings.